app/src/main/res/drawable/ic_home.xml

```diff
@@ -1,14 +1,12 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we not want this at the top of the drawable files?

Reply:
Oops we do; updating